### ISSUE TITLE
Newsup.pl not working in Windows 10 Environment

### ISSUE THREAD

**Issue by cavalia88** [demanuel/NewsUP#24]
As mentioned, for some undetermined reason, Newsup.pl does not work in a Windows 10 environment.

Using Activestate Perl to run newsup.pl, program will not even run and a series of errors will be thrown out

Using Strawberry Perl to run newsup.pl, program will run but extremely slowly (i.e. much slower than in Windows 7). Upload speeds on both SSL and non-SSL ports equally slow. Will also throw up errors when using header check function.

Till this is sorted out, suggest all users to avoid using newsup.pl in Windows 10 environment

**Comment by demanuel** [comment]
Unfortunately, i don't a have a windows 10. To test it. 
I need to get a windows 10 environment to test. Since this is developed in Linux, i can guarantee that it will work on it.

**Comment by demanuel** [comment]
Hi!

On the [official windows 10 VM]( https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/linux/) I was able to run it and reach similar velocities as the ones i get in linux.

**I was unable to use strawberry perl 5.022.** I used strawberry perl 5.020.  Because of this i had to use on the code `use 5.020;`

The Inline::C and Config::Tiny were missing in strawberry perl so i had to run on the CMD line:

- cpan  install Inline::C
- cpan install Config::Tiny
